provider/pkg/provider/helm_release.go

```diff
@@ -348,7 +347,7 @@ func (r *helmReleaseProvider) Check(ctx context.Context, req *pulumirpc.CheckReq
 		KeepSecrets:  true,
 	})
 	if err != nil {
-		return nil, pkgerrors.Wrapf(err, "check failed because malformed resource inputs: %+v", err)
+		return nil, fmt.Errorf("check failed because malformed resource inputs: %+v: %w", err, err)
```

Comment:
Seems unintentional that `err` is used twice.

Reply:
Good catches! This was duplicating the error and my script preserved the behavior because it's just another formatting arg. The original author might have forgotten or not realized `Wrapf` already did this automatically.